submissions/Kostiantyn_Rakhmanov/hooli-style-popup/index.html

```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hooli-style Popup</title>
+    <link rel="stylesheet" href="./styles.css">
+</head>
+<body>
+    <div class="wrapper">
+        <header class="header">
+            <nav class="menu">
+                <ul class="menu__items">
+                    <li class="menu__item"><a class="menu__link" href="#">Mail</a></li>
+                    <li class="menu__item"><a class="menu__link" href="#">Images</a></li>
+                    <li class="menu__item"><a class="menu__link" href="#"></a>
+                        <input class="products__checkbox" type="checkbox" id="products__checkbox">
+                        <label for="products__checkbox" class="products__check-label">
+                            <img class="products-img" src="./img/popup-button.png" alt="#">
+                        </label>
```

Comment:
Avoid shifting sibling elements on hover or focus. Also, fix styles on focus. [Watch video]  (https://share.vidyard.com/watch/1h9zWKjLxTb9phZeHgXu6P?) All menu items must be accessible from the keyboard. 
Add `cursor: pointer` to all interactive elements.